mllib/src/main/scala/org/apache/spark/ml/classification/LogisticRegression.scala

```diff
@@ -803,13 +829,17 @@ private class LogisticAggregator(
     this
   }
 
-  def count: Long = totalCnt
-
-  def loss: Double = lossSum / totalCnt
+  def loss: Double = {
+    require(weightSum >= 1.0, s"The effective number of samples should be " +
```

Comment:
When computing the variance, there is no definition of variance when 0.0 < # of sample < 1.0